src/documentation/0001-node-introduction/index.md

```diff
@@ -38,7 +38,7 @@ The most common example Hello World of Node.js is a web server:
 const http = require('http')
 
 const hostname = '127.0.0.1'
-const port = 3000
+const port = 3000 || process.env.PORT
```

Comment:
This is backwards, as 3000 will never be falsey

Reply:
```suggestion
const port = process.env.PORT || 3000